qmail-qmqpc.c

```diff
@@ -135,7 +135,8 @@ char *server;
 
 stralloc servers = {0};
 
-main()
+void
```

Comment:
I personally use this style, but this has the side effect of shifting the line numbers (does it actually matter for patch?) and is not consistent with the rest of the source, with type and function name on the same line.

Reply:
The line number change will not harm, patch will just output a message about offset.